### PR TITLE
DropDown from Home tag is removed.

### LINKED ISSUE #56: 🐞 [BUG_LP] - Removing dropdown from home button
### Description of the bug. ✍🏻

In home button of navigation bar there is a drop down with several options named Home 1, Home 2,..... . And they are redirecting to home only . I believe that there is no need of the drop down, so with to remove the dropdown from home button.

### Current Behaviour

Currently home button is having a drop down with several options redirecting to home page only

### Expected Behaviour

Home button will not have any dropdown option

### Reproduction Steps

```bash
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
```

### Screenshots

Screenshot 2023-05-31 at 2 12 25 PM

### Additional information

_No response_

### Contributor terms

- [X] I agree to follow the Contributing Instructions

### PR DESCRIPTION
**Pull request by Khushisharma23**
## Related Issue(s)

 It Closes #56 

## Proposed Changes
 Home Tag had an unwanted dropdown menu so I removed it.

## Type of Change

- [ ] New Feature (Landing Page)
- [ ] New Feature (App)
- [x] Bug (Landing Page)
- [ ] Bug (App)
- [ ] New Component (App)
- [ ] Enhancement
- [ ] Documentation / ReadMe / Workflows
- Other (If not in above options):  

## Your Idea for Level (GSSoC)

- [ ] Level 1
- [x] Level 2
- [ ] Level 3

## Screenshots (if applicable)
 ###Before Changes
image
###After Changes
image

### CODE REVIEW

**Review by praveenscience — CHANGES_REQUESTED** [review]
Can you get rid of the `.vscode` folder?

**Comment by praveenscience** [comment]
Use `prettier-width: 800000` as well, thanks! 😊

**Comment by Khushisharma23** [comment]
Yup let me remove it

File: index.html [hunk 1]
```diff
@@ -1550,9 +1550,6 @@ <h5>Our Headquarters</h5>
                 <ul class="list-unstyled social-list mb-0">
                   <li class="list-inline-item">
 
-                 
-
-                    <a href="#" class="rounded" aria-label="Visit us on Facebook" title="Facebook (External Link)" rel=”noopener noreferrer”><span class="ti-facebook white-bg color-2 shadow rounded-circle"></span></a>
                   </li>
```
Comment (praveenscience) [inline index.html]: Why is there an empty tag here?

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Comment by Khushisharma23** [comment]
This PR is totally ruined. Sorry! :)

**Comment by praveenscience** [comment]
Failure is the stepping stone to success! 💪

**Pull request closed without merging**